test/bms-test.c

```diff
@@ -0,0 +1,21 @@
+/*********************************** Macros ****************************************/
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main()
+
+
+
+/*********************************** Includes **************************************/
+#include "catch.hpp"
+#include "bms-data.h"
+
+bool isStopRequestedByUser = false;
+
+
+/*********************************** Test Cases **************************************/
+
+TEST_CASE("printed value is in range") {
+    BmsDataSendToConsole();
+    REQUIRE((getBmsTemperatue() >= TEMPERATURE_MIN));
+    REQUIRE((getBmsTemperatue() <= TEMPERATURE_MAX));
```

Comment:
These are random numbers checked once to be in range. They may pass even if the ranges are wrong. What is the value of these tests?
Hint: You are testing random number generation here, which is a functionality of the runtime. Test your functionality.